tools/lsp/preview.rs

```diff
@@ -117,6 +117,22 @@ fn lsp_element_position(element: &ElementRc) -> (String, lsp_types::Range) {
         .unwrap_or_default()
 }
 
+// triggered from the UI, running in UI thread
```

Comment:
It would probably be useful to have some naming convention or something so such comments are not needed. Perhaps split into modules with one module for the UI thread, and a module for the LSP or something like that. (Don't need to be part of this PR)